src/main/java/net/dv8tion/jda/internal/JDAImpl.java

```diff
@@ -715,8 +714,6 @@ public synchronized void shutdownInternals()
         // stop accepting new requests
         if (requester.stop()) // returns true if no more requests will be executed
             shutdownRequester(); // in that case shutdown entirely
-        if (audioLifeCyclePool != null)
-            audioLifeCyclePool.shutdownNow();
```

Comment:
Shouldn't this still shutdown the pool at some point when its default? Thats what `ThreadConfig#isShutdownAudioPool` is used for.

Reply:
I'm not 100% sure what you mean here, can you elaborate?

I think this change, removing these two lines, is safe, because the next line, `threadConfig.shutdown()`, takes care of shutting down the audio pool now. There is only a small difference of behaviour, in that `threadConfig.shutdown()` triggers `shutdown()` instead of `shutdownNow()`, but I don't think that makes a difference.

What do you think?